### DISCUSSION TITLE
ErrorException: Undefined array key "id"

### DISCUSSION THREAD

**Discussion by brocorp-dev** [livewire/livewire#3745] [7. Bugs (archived)] [answered]
Hi guys,

We encountered an error since we upgrading to 2.6.2 this morning on local dev and staging server

PHP version : 8.0.2
Laravel version : 8.61.0
Octane version : 1.0.11
Swoole version : 4.6.6

On our full page components, with or without Octane, we occurs systematically this error when we call any action : 

ErrorException: Undefined array key "id" 
vendor/livewire/livewire/src/HydrationMiddleware/PerformActionCalls.php:27

![image](https://user-images.githubusercontent.com/46473786/133413919-0b8f443e-3c1c-4b0b-b583-6d3a62504436.png)

**Comment by joshhanley** [comment, marked as answer]
@brocorp-dev do you publish Livewire's javascript assets? If so can you try and publish them again and see if that fixes it?

**Reply by brocorp-dev** [reply]
Hi @joshhanley, 

We republished Livewire Javascript assets with php artisan livewire:publish --assets and we don't encountered anymore this issue 😎

Thanks for your help!

**Reply by joshhanley** [reply]
@brocorp-dev awesome! No worries 🙂

You can setup a composer script to do it automatically, have a look at the "Publishing Frontend Assets" section here for instructions https://laravel-livewire.com/docs/2.x/installation
